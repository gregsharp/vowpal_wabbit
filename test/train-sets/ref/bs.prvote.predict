-1.000000 0.458542 1.000000
-1.000000 0.473533 0.607041
-1.000000 0.340097 0.484781
-1.000000 0.209609 0.349782
-1.000000 0.259589 0.402866
-1.000000 0.885245 0.977926
-1.000000 0.289948 0.454686
-1.000000 0.110648 0.436562
-1.000000 0.334399 0.461862
-1.000000 0.784682 1.000000
-1.000000 0.269292 0.393562
-1.000000 0.336200 0.568953
-1.000000 0.215788 0.458039
-1.000000 0.337985 0.570398
-1.000000 0.740010 0.809205
-1.000000 0.578930 0.753161
-1.000000 0.889724 1.000000
-1.000000 0.213896 0.403085
-1.000000 0.224411 0.644630
-1.000000 0.053549 0.203826
-1.000000 0.258511 0.753945
-1.000000 0.590986 1.000000
-1.000000 0.135984 0.314393
-1.000000 0.478131 0.939110
-1.000000 0.000000 0.182336
-1.000000 0.217458 0.452702
-1.000000 0.201567 0.446333
-1.000000 0.104638 0.329909
-1.000000 0.483572 0.969659
-1.000000 0.244790 0.439700
-1.000000 0.661956 1.000000
-1.000000 0.204253 0.435376
-1.000000 0.132103 0.218262
-1.000000 0.251499 0.423116
-1.000000 0.619113 0.980061
-1.000000 0.193126 0.562957
-1.000000 0.634806 1.000000
-1.000000 0.121480 0.323060
-1.000000 0.557163 1.000000
1.000000 1.000000 1.000000
-1.000000 0.235915 0.359089
-1.000000 0.752914 0.841858
-1.000000 0.072550 0.159824
-1.000000 0.130488 0.371797
-1.000000 0.102583 0.319911
-1.000000 0.123988 0.229874
-1.000000 0.295392 0.659599
-1.000000 0.179231 0.500906
-1.000000 0.341425 0.841522
-1.000000 0.199805 0.278299
-1.000000 0.480692 0.904947
1.000000 0.397531 1.000000
-1.000000 0.000000 0.417197
-1.000000 0.123438 0.460271
-1.000000 0.877451 1.000000
-1.000000 0.354258 0.829364
-1.000000 0.125584 0.304976
-1.000000 0.084983 0.374619
-1.000000 0.373036 0.912963
-1.000000 0.051437 0.421569
-1.000000 0.574689 0.894669
-1.000000 0.034918 0.409091
-1.000000 0.477929 0.788760
-1.000000 0.073380 0.266271
-1.000000 0.606537 1.000000
-1.000000 0.101940 0.355618
-1.000000 0.154540 0.412049
-1.000000 0.174071 0.719000
-1.000000 0.056280 0.190277
-1.000000 0.543227 0.740439
-1.000000 0.109307 0.418877
-1.000000 0.457044 0.787100
-1.000000 0.312281 0.824736
-1.000000 0.066846 0.271112
-1.000000 0.717398 0.934371
-1.000000 0.431120 0.952748
-1.000000 0.055533 0.397098
-1.000000 0.206126 0.619509
-1.000000 0.043797 0.112990
-1.000000 0.072012 0.345589
0.000000 0.000000 0.218369
-1.000000 0.139673 0.377842
-1.000000 0.806658 1.000000
-1.000000 0.124006 0.465731
-1.000000 0.151144 0.210641
-1.000000 0.103022 0.364594
-1.000000 0.333333 0.959691
-1.000000 0.624276 1.000000
1.000000 0.716232 1.000000
-1.000000 0.000000 0.337187
-1.000000 0.081438 0.204202
-1.000000 0.483797 0.843803
1.000000 0.959783 1.000000
-1.000000 0.077385 0.176741
-1.000000 0.629873 1.000000
-1.000000 0.010733 0.256136
-1.000000 0.507555 0.989902
-1.000000 0.057807 0.282666
-1.000000 0.410453 0.869668
-1.000000 0.492611 0.983345
-1.000000 0.106847 0.485936
-1.000000 0.544805 0.906455
-1.000000 0.000000 0.121606
-1.000000 0.469715 0.980938
-1.000000 0.102511 0.246307
-1.000000 0.921458 1.000000
-1.000000 0.000000 0.141087
-1.000000 0.044143 0.354397
-1.000000 0.097736 0.253467
1.000000 0.917709 1.000000
-1.000000 0.604574 1.000000
-1.000000 0.211312 0.589191
-1.000000 0.000000 0.414538
-1.000000 0.359269 0.932506
-1.000000 0.076806 0.419534
-1.000000 0.000000 0.523672
-1.000000 0.438833 0.759181
-1.000000 0.472129 1.000000
-1.000000 0.367939 0.999147
-1.000000 0.011941 0.524531
-1.000000 0.031902 0.360733
-1.000000 0.770982 0.894703
-1.000000 0.052012 0.436470
-1.000000 0.471281 0.949459
-1.000000 0.498583 1.000000
-1.000000 0.356853 0.778179
-1.000000 0.000000 0.406344
-1.000000 0.394071 0.919628
-1.000000 0.000000 0.395337
-1.000000 0.411538 0.918851
-1.000000 0.032771 0.154788
1.000000 0.695526 1.000000
-1.000000 0.040298 0.262463
-1.000000 0.251122 0.851623
-1.000000 0.042627 0.369002
-1.000000 0.023765 0.337782
-1.000000 0.499659 1.000000
-1.000000 0.359683 0.894069
-1.000000 0.864957 0.958315
-1.000000 0.019719 0.388124
-1.000000 0.077161 0.449558
-1.000000 0.000000 0.216000
-1.000000 0.568520 0.942772
-1.000000 0.751802 0.990763
-1.000000 0.449324 0.935751
-1.000000 0.511314 0.805219
-1.000000 0.470428 0.939698
1.000000 0.831640 1.000000
0.000000 0.000000 0.355941
-1.000000 0.318933 0.927077
-1.000000 0.790148 1.000000
-1.000000 0.232560 0.924183
-1.000000 0.548524 0.939848
-1.000000 0.080518 0.400425
-1.000000 0.003844 0.651519
-1.000000 0.521165 0.946280
-1.000000 0.866355 1.000000
-1.000000 0.000000 0.055260
-1.000000 0.687787 0.985260
-1.000000 0.006398 0.539823
-1.000000 0.497750 0.840301
0.000000 0.000000 0.221798
-1.000000 0.013456 0.422433
-1.000000 0.638838 0.856537
-1.000000 0.056091 0.325332
-1.000000 0.354644 0.819553
-1.000000 0.951429 0.989244
-1.000000 0.000000 0.196094
-1.000000 0.463253 0.991503
-1.000000 0.698086 0.997982
-1.000000 0.524908 0.782291
-1.000000 0.105022 0.398213
-1.000000 0.000000 0.323968
-1.000000 0.328826 0.655503
-1.000000 0.075495 0.260934
-1.000000 0.066262 0.605312
-1.000000 0.002131 0.319530
-1.000000 0.746377 0.976163
-1.000000 0.652179 0.978861
-1.000000 0.684518 0.921459
-1.000000 0.446896 0.863388
-1.000000 0.147498 0.241931
-1.000000 0.415937 0.813590
-1.000000 0.000000 0.306481
-1.000000 0.039986 0.423755
-1.000000 0.000000 0.524194
-1.000000 0.568177 1.000000
-1.000000 0.087538 0.568334
-1.000000 0.029076 0.362352
-1.000000 0.665350 0.936338
-1.000000 0.646028 0.844182
-1.000000 0.052672 0.337844
0.000000 0.000000 0.023396
-1.000000 0.012676 0.308387
-1.000000 0.125693 0.359239
-1.000000 0.981854 1.000000
-1.000000 0.441630 0.910354
-1.000000 0.000000 0.008660
-1.000000 0.014539 0.469021
-1.000000 0.567479 0.962731
