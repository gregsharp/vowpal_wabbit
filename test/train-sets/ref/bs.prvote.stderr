only testing
Num weight bits = 18
learning rate = 10
initial_t = 1
power_t = 0.5
predictions = bs.prvote.predict
using no cache
Reading datafile = train-sets/0001.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
1.000000   1.000000            1         1.0   1.0000  -1.0000       51
1.000000   1.000000            2         2.0   0.0000  -1.0000      104
1.000000   1.000000            4         4.0   0.0000  -1.0000      135
1.000000   1.000000            8         8.0   0.0000  -1.0000      146
1.000000   1.000000           16        16.0   1.0000  -1.0000       24
1.000000   1.000000           32        32.0   0.0000  -1.0000       32
0.968750   0.937500           64        64.0   0.0000  -1.0000       61
0.953125   0.937500          128       128.0   1.0000  -1.0000      106

finished run
number of examples per pass = 200
passes used = 1
weighted example sum = 200
weighted label sum = 0
average loss = 0.945
best constant = -0.00502513
total feature number = 15482
